py/specter/util/util.py

```diff
@@ -242,4 +242,135 @@ def legval_numba(x, c):
         c0 = c[-i] - (c1*(nd - 1))*nd_inv
         c1 = tmp + (c1*x*(2*nd - 1))*nd_inv
     return c0 + c1*x
+  
+
+@numba.jit(nopython=True, cache=False)
+def custom_hermitenorm(n, u):
+    #below is (mostly) cut and paste from scipy orthogonal_eval.pxd
+    #some modifications have been made to operate on an array
+    #rather than a single value (as in the original version)
+    res=np.zeros(len(u))
+    if n < 0:
+        return (0.0)*np.ones(len(u))
+    elif n == 0:
+        return (1.0)*np.ones(len(u))
+    elif n == 1:
+        return u
+    else:
+        y3 = 0.0
+        y2 = 1.0
+        for i,x in enumerate(u):
+            for k in range(n, 1, -1):
+                y1 = x*y2-k*y3
+                y3 = y2
+                y2 = y1
+            res[i]=x*y2-y3
+            #have to reset before the next iteration
+            y3 = 0.0
+            y2 = 1.0
+        return res
+
+
+@numba.jit(nopython=True, cache=False)
+def custom_erf(y):
```

Comment:
Please add a docstring mentioning that this is functionally equivalent to `scipy.erf(y)` and whether it requires array input or if it will also work with scalars.  I think it requires array input, but that should be documented for the user in the docstring without having read the code.

Having gone through the pain of writing this, I now see that there is also `numpy.erf`, which numba probably would have supported.  This appears to be ~15% faster so ok to keep.

Please include URL to location of scipy erf.f for reference.

Reply:
Docstring added. Note about supporting arrays but not scalars also included. Link to original scipy erf also included. 

Excellent point about numpy.erf. Although documentation is spotty, it does work in Numba nopython=False mode (but is substantially slower, runtimes 3:40 vs 2:20 for Haswell). It doesn't appear to be officially supported by Numba at this time but this may change going forward.